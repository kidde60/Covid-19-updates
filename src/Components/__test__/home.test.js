import renderer from 'react-test-renderer';

describe('Currency display container', () => {
  it('Confirm that the currency display renders correctly', () => {
    const tree = renderer.create(<fetchCountries />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
