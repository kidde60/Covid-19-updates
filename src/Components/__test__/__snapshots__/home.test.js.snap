// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Currency display container Confirm that the currency display renders correctly 1`] = `<fetchCountries />`;
